labs/bdls.md

```diff
@@ -0,0 +1,30 @@
+---
+layout: default
+title: BDLS
+[//]: # (parent: Labs)
+---
+# Lab Name
+BDLS
+
+# Short Description
```

Comment:
The short description is what will be used within a GitHub's project's About section. Can we shorten this? The information is interesting, and should be included in the Scope of the Lab, but we will pull the information here when setting up the project in GitHub.

Reply:
Sure, short description has been replaced